edith-step/src/components/ReceiptInput.js

```diff
@@ -18,99 +33,133 @@ export default class ReceiptInput extends React.Component {
+              <div className="input-group mb-2">
+                <div className="input-group-prepend">
+                  <div className="input-group-text">Item</div>
+                </div>
+                <input
+                  type="text"
+                  className="form-control"
+                  name="itemName"
+                  id="name"
+                  value={this.state.itemName}
+                  onChange={this.handleChange} />
+              </div>
+            </div>
+            <div className="col-auto">
+              <div className="input-group mb-2">
+                <div className="input-group-prepend">
+                  <div className="input-group-text">Price</div>
+                </div>
+                <input
+                  type="number"
+                  className="form-control"
+                  name="itemPrice"
+                  id="price"
+                  step="0.01"
+                  value={this.state.itemPrice}
+                  onChange={this.handleChange} />
+              </div>
+            </div>
+            <div className="col-auto">
+              <div className="input-group mb-2">
+                <div className="input-group-prepend">
+                  <div className="input-group-text">Quantity</div>
+                </div>
+                <input
+                  type="number"
+                  className="form-control"
+                  name="itemQuantity"
+                  id="quantity"
+                  step="1"
+                  value={this.state.itemQuantity}
+                  onChange={this.handleChange} />
+              </div>
+            </div>
+            <div className="col-auto">
+              <button className="btn btn-primary"
                 id="submit"
-                type="submit" 
-                value="Submit" />
-            </td>
-          </tr>
-        </tbody>
-        </table>
-      </form>
+                type="submit"
+                value="Submit">Add Item</button>
+            </div>
+          </div>
+        </form>
+        {this.state.items.length > 0 &&
+        <GroceryList items={this.state.items} />
+        }
       </div>
     );
   }
 }
 
 const GroceryList = (props) => {
   return (
-    <table id="grocery-list">
-      <tbody>
-        {props.items.map(item => (
-          <tr className="item" key={item.id}>
-            <td className="item-name">{item.itemName}</td>
-            <td className="item-price">{item.itemPrice}</td>
-            <td className="item-quantity">{item.itemQuantity}</td>
-          </tr>
+    <div id="grocery-list">
+      <ul className="list-group col-lg-3">
+        <li className=
+          "h-50 list-group-item d-flex justify-content-between align-items-center">
+          <span className="col-lg-1">Item</span>
+          <span className="badge badge-pill col-lg-1">Price</span>
+          <span className="badge badge-pill col-lg-1">#</span>
+        </li>
+        {props.items.map((item) => (
```

Comment:
Lint error

> 'items' is missing in props validationeslint(react/prop-types)

Please fix.

See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md#prevent-missing-props-validation-in-a-react-component-definition-reactprop-types for more information about the error.

Possible replacement:

```jsx
const GroceryList = createReactClass({
  propTypes: {
    items: PropTypes.arrayOf(PropTypes.object), // or more specific if possible
  },
  render() {
    const props = this.props; 
    ... 
  },
});
```